### PR TITLE
Feature: Union schema compatibility

### PR DESCRIPTION
**Pull request by fivetran-catfritz**
Issue: 
- https://github.com/fivetran/dbt_app_reporting/issues/20

Confirm the following files were correctly updated automatically:
- [x] CHANGELOG
- [x] README
- [x] google_play.yml
- [x] docs.md
- [x] joins
- [x] window functions (partition by)
- [x] 'source_relation' column added
- [x] tests

Manual updates:
- [x] Add `source_relation` to downstream models if necessary
- [x] Finish any incomplete/incorrect joins/partitions
- [x] Update tests to include `source_relation` in unique-combination-of-cols if necessary
    - May need to remove some uniqueness tests in favor of the combo test

Validation:
- When not using unioning, tests in the folder `tests/consistency/row_counts` pass
  - It is is expected that the consistency tests will not pass when unioning
- Spot check with seed data for unioning:
  - Screenshot 2024-07-18 at 1 08 50 PM

- Also:
  - see internal ticket

### CODE REVIEW

File: packages.yml [hunk 1]
```diff
@@ -1,3 +1,6 @@
 packages:
-- package: fivetran/google_play_source
-  version: [">=0.3.0", "<0.4.0"]
\ No newline at end of file
+# - package: fivetran/google_play_source
+#   version: [">=0.3.0", "<0.4.0"]
+- git: https://github.com/fivetran/dbt_google_play_source.git
+  revision: MagicBot/add-union-schema
+  warn-unpinned: false
```
Comment (fivetran-joemarkiewicz) [inline packages.yml:6]: Reminder to update before release

File: CHANGELOG.md
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:1]: Please apply the same updates to from the Apple PRs to this CHANGELOG.

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
@fivetran-catfritz this PR is looking great, just a small request to update the CHANGLOG with the same notes from the Apple Store PRs.

**Comment by fivetran-joemarkiewicz** [comment]
Also before merging we should add the auto releaser to this package.

**Review by fivetran-catfritz — COMMENTED** [review]
@fivetran-joemarkiewicz I have updated, including the changes from apple_store. This is ready for re-review!

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM thanks for applying the updates!

**Pull request merged**